test/common-build-steps.ts

```diff
@@ -198,21 +198,21 @@ function setData(Data calldata d) external;`,
 import { Example2Facet } from "../facets/Example2Facet.sol";`,
           __NUM_FACETS__: '2',
           __CUTS__: `
-bytes4[] memory f0 = new bytes4[](2);
-f0[0] = IDiamondProxy.getInts.selector;
-f0[1] = IDiamondProxy.getData.selector;
+bytes4[] memory f = new bytes4[](2);
+f[0] = IDiamondProxy.getInts.selector;
+f[1] = IDiamondProxy.getData.selector;
 cut[0] = IDiamondCut.FacetCut({
   facetAddress: address(new ExampleFacet()),
   action: IDiamondCut.FacetCutAction.Add,
-  functionSelectors: f0
+  functionSelectors: f
 });
 
 bytes4[] memory f1 = new bytes4[](1);
```

Comment:
This doesn't seem quite right - why are we initialising `f1` if we don't use it below? Also, does this work, i.e. can we re-initialise `f` to a new dynamic array and everything is copied into the cut struct?

Reply:
just pushed a fix, test is passing now:
```
  Command: build() - Foundry
    with multiple facets and methods with custom structs
      ✔ generates test helper (2571ms)


  1 passing (3s)
```